extensions/wikia/AdEngine/js/lookup/openx/openXBidderHelper.js

```diff
@@ -0,0 +1,93 @@
+/*global define*/
+/*jshint camelcase:false*/
+define('ext.wikia.adEngine.lookup.openx.openXBidderHelper', [
```

Comment:
I wasn't sure where to put this file. I decided to place it here although I am open for any suggestions.